test/e2e/datadogagentdeployment_test.go

```diff
@@ -8,29 +8,30 @@ package e2e
 import (
 	goctx "context"
 	"fmt"
+	"io"
 	"io/ioutil"
 	"path/filepath"
 	"testing"
 	"time"
 
-	"k8s.io/client-go/kubernetes/scheme"
-
 	"github.com/DataDog/datadog-operator/pkg/apis"
 	datadoghqv1alpha1 "github.com/DataDog/datadog-operator/pkg/apis/datadoghq/v1alpha1"
-
 	"github.com/DataDog/datadog-operator/test/e2e/utils"
+
 	framework "github.com/operator-framework/operator-sdk/pkg/test"
 	"github.com/operator-framework/operator-sdk/pkg/test/e2eutil"
-	rbacv1 "k8s.io/api/rbac/v1"
-	dynclient "sigs.k8s.io/controller-runtime/pkg/client"
 
 	appsv1 "k8s.io/api/apps/v1"
 	corev1 "k8s.io/api/core/v1"
+	rbacv1 "k8s.io/api/rbac/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes/scheme"
+	dynclient "sigs.k8s.io/controller-runtime/pkg/client"
 )
 
 var (
 	retryInterval        = time.Second * 5
-	timeout              = time.Second * 60
+	timeout              = time.Second * 120
```

Comment:
I believe this change was for testing purpose right?

Reply:
in some case 120 second maybe require like: rolling update of a new daemonset.
120sec seems not so high